### PR TITLE
Adding cloud target guidance

### PR DESCRIPTION
**Pull request by blckmn**

### CODE REVIEW

**Review by sugaarK — DISMISSED** [review]

**Review by KarateBrot — DISMISSED** [review]

File: docs/development/manufacturer/config-target-guidance.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Config Target Guidance - Cloud Build
```
Comment (KarateBrot) [inline docs/development/manufacturer/config-target-guidance.md:1]: to make it explicitly the first one you can add this at the top of the file. But it needs to be a `.mdx` file afaik (not just `.md`)

```mdx
---
sidebar_position: 1
---
```

Comment (nerdCopter) [reply docs/development/manufacturer/config-target-guidance.md:1]: confirmed.
mv to .mdx & add the above as line 1-3.
image

File: docs/development/manufacturer/config-target-guidance.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Due to the increase in poorly designed flight controllers, primarily clones of known poorly designed units, Betaflight is now taking a strong stance on adding targets to our cloud build platform.
+
+From the 1st of July 2024 **ALL** targets presented to the config repository for use in the cloud build system will be required to follow these conditions:
+
+
+1. Schematics need to be presented with the prospective config file. If your company requires these schematics to not be public they can be emailed to [cloudtargets@betaflight.com](mailto:cloudtargets@betaflight.com)
+
+
+2. Our team will assess the schematics. If the schematic has resource sharing, lack of DMA and or timers on critical functions eg gyro, motors, sharing SPI bus lines or is a blatant clone of existing designs (especially so if we consider it to be poorly designed), it will be DENIED a target until such time as it is redesigned to both meet the expected performance of the betaflight firmware, and demonstrates it is not merely a clone of an existing design.
```
[the remark this answers is not in the document]
Comment (blckmn) [reply docs/development/manufacturer/config-target-guidance.md]: Done

File: docs/development/manufacturer/config-target-guidance.md (hunk 1 above)
Comment (blckmn) [reply docs/development/manufacturer/config-target-guidance.md]: Done.

**Review by sugaarK — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**